.gitignore

```diff
@@ -102,7 +102,6 @@ ENV/
 
 # molecule
 roles/**/molecule/**/__pycache__/
-roles/**/molecule/**/*.conf
```

Comment:
Why do we need to remove this line?

Reply:
I needed to add a cni definition file which has the *.conf extension, `10-mynet.conf` and this was prevented by this .gitignore rule. It's not clear to me why this was added in the first place since .conf files are valid objects in molecule tests.

Reply:
I checked the pull request https://github.com/kubernetes-sigs/kubespray/pull/8080 which added the above line.
But I cannot find the reason.
It is fine to remove it for me.

/lgtm